tests/TestCase/Shell/BakeShellTest.php

```diff
@@ -87,7 +87,7 @@ public function testAllWithModelName()
             ->with('Comments');
 
         $this->Shell->connection = '';
-        $this->Shell->params = [];
+        $this->Shell->params = ['prefix' => 'account'];
```

Comment:
@antograssiot If the prefix param was passed before this PR, an exception would rise referring an invalid param. The new prefix option added to the parser covers it.